packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -538,12 +538,25 @@ class TextSelectionOverlay {
   }
 
   void _handleSelectionHandleChanged(TextSelection newSelection, {required bool isEnd}) {
-    final TextPosition textPosition = isEnd ? newSelection.extent : newSelection.base;
+    final TextPosition bringTextPosition;
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.iOS:
+      case TargetPlatform.macOS:
+        bringTextPosition = newSelection.extent;
```

Comment:
This is because MacOS/iOS always make the dragged handle the extentHandle (https://github.com/flutter/flutter/pull/105069), right?  Maybe add a comment about that.